EasyCommands.Tests/EasyCommands.Tests.csproj

```diff
@@ -173,11 +173,16 @@
     <Compile Include="ScriptTests\FunctionalTests\ControlCommandTests.cs" />
     <Compile Include="ScriptTests\FunctionalTests\IterationCommandTests.cs" />
     <Compile Include="ScriptTests\FunctionalTests\SimpleWaitCommandTests.cs" />
-    <Compile Include="ScriptTests\FunctionalTests\SimpleComparisonTests.cs" />
+    <Compile Include="ScriptTests\FunctionalTests\SimpleComparisonsOtherTests.cs" />
```

Comment:
I'm hoping we can just get rid of these tests, presuming they are covered by the new tests.

Reply:
This file contains the test for the ternary and for colors